Authentication/SignUp/View/SignupViewDelegate.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  SignupViewDelegate.swift
+//  Authentication
+//
+//  Created by Daniela Riesgo on 4/1/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+public protocol SignupViewDelegate {
+    
+    var colourPalette: ColorPaletteType { get }
+    var fontPalette: FontPaletteType { get }
+    
+    func configureView(signupView: SignupViewType)
+    
+}
+
+public extension SignupViewDelegate {
+    
+    func configureView(signupView: SignupViewType) {
+        
+    }
+    
+}
+
+public final class DefaultSignupViewDelegate: SignupViewDelegate {
+    
+    public let colourPalette: ColorPaletteType
+    public let fontPalette: FontPaletteType
+    
+    public init() {
```

Comment:
I would change this init for receiving the palettes as dependencies (and having them with default values)